packages/devtools/src/getMonolithicCSSRules.ts

```diff
@@ -1,7 +1,8 @@
 import * as stylis from 'stylis';
 import type { AtomicRules, MonolithicAtRules, MonolithicRules, RuleDetail } from './types';
 
-const griffelClassNameRegex = /^\.f[0-9a-z]+$/;
+// match className like '.fvnxzrg' or '.fvnxzrg.fui-focus-visible'
+const griffelClassNameRegex = /^\.(f[0-9a-z]+)(\..*|$)/;
```

Comment:
Will it handle `body .fvnxzrg`?

Reply:
Yeah it should, I have a test with rules like `'[data-keyboard-nav] .fabcde1{color:red;}'`

Reply:
The parser will try to find the griffel className, taken it out and leave the rest: `body .fvnxzrg` -> `body `